src/mlx/jira_juggler/jira_juggler.py

```diff
@@ -594,8 +594,36 @@ def load_issues_from_jira(self, depend_on_preceding=False, sprint_field_name='',
             try:
                 issues = jirahandle.search_issues(self.query, maxResults=JIRA_PAGE_SIZE, startAt=self.issue_count,
                                                   expand='changelog')
-            except JIRAError:
-                logging.error('Invalid Jira query "%s"', self.query)
+            except JIRAError as err:
+                logging.error('Failed to query JIRA: %s', err)
```

Comment:
```suggestion
                logging.error(f'Failed to query JIRA: {err}')
```
f-string formatting was included in Python 3.6 (made better in Python 3.12)
https://peps.python.org/pep-0498/

Reply:
How was it made better in Python 3.12?